### PR TITLE
added basic button embeds with discord messages

### PR DESCRIPTION
**Pull request by Sudowuko**

### CODE REVIEW

File: discord.log [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[2022-07-29 18:58:48] [INFO    ] discord.client: logging in using static token
```
Comment (jamesnicolas) [inline discord.log]: don't commit this file

**Review by jamesnicolas — APPROVED** [review]

**Comment by Sudowuko** [comment]
@jamesnicolas can you review this? I finished added the commands for tokens, quest count, logs, team name and habit

I remember you said I shouldn't just merge my code without someone looking at it first lol

File: main.py [hunk 2]
```diff
@@ -4,6 +4,13 @@
 from firebase_admin import credentials
 from firebase_admin import firestore
 from discord.ext import commands
+from discord.utils import get
+# from discord_components import DiscordComponents, Button, Select, SelectOption, Component
```
Comment (jamesnicolas) [inline main.py]: remove this

File: main.py [hunk 3]
```diff
@@ -20,64 +27,186 @@
 
 intents = discord.Intents.default()
 intents.members = True
+intents.message_content = True
+
+sudo = commands.Bot(intents=intents, command_prefix='!')
+ACount = 0
```
Comment (jamesnicolas) [inline main.py]: clean this up

File: main.py [hunk 4]
```diff
@@ -50,34 +100,23 @@ async def addToken(ctx, arg):
...
+    doc_ref = db.collection('users').document(str(user.id))
+    tokens = doc_ref.get().get("tokens")
+    print("ctx is: ", ctx)
+    await ctx.send(f"{user} currently has {tokens} tokens")
+
+@sudo.command()
 async def checkUsername(ctx):
     doc_ref = db.collection('users').document(str(ctx.author.id))
     username = doc_ref.get().get("username")
+    print("ctx is: ", ctx)
```
Comment (jamesnicolas) [inline main.py:204]: remove this

File: main.py [hunk 5]
```diff
@@ -20,64 +27,186 @@
...
+    doc_ref.set({
+        'tokens': int(arg1),
+        'quests': int(arg2),
+        'mlogs': int(arg3),
+        'team': arg4,
+        'habit': arg5,
+    })
+    tokens = doc_ref.get().get("tokens")
+    quests = doc_ref.get().get("quests")
+    mlogs = doc_ref.get().get("mlogs")
```
Comment (jamesnicolas) [inline main.py]: mlogs is to vague of a name, choose something like “monthly_logs”

**Pull request merged**